x/lsnative/staking/keeper/slash.go

```diff
@@ -131,7 +131,12 @@ func (k Keeper) Slash(ctx sdk.Context, consAddr sdk.ConsAddress, infractionHeigh
 	validator = k.RemoveValidatorTokens(ctx, validator, tokensToBurn)
 
 	// Proportionally deduct any liquid tokens from the global total
-	validatorLiquidRatio := validator.TotalLiquidShares.Quo(validator.DelegatorShares)
+	var validatorLiquidRatio sdk.Dec
+	if validator.TotalLiquidShares.IsNil() {
```

Comment:
Do we need this check in other places, (for ex. [here](https://github.com/persistenceOne/persistence-sdk/blob/b/fix-validator-state/x/lsnative/staking/keeper/liquid_stake.go#L77))

And also for `TotalValidatorBondShares`?

Reply:
Won't need this check too, after state is properly migrated. This change only to skip the failing block and move on

Migration is there https://github.com/persistenceOne/persistenceCore/pull/195/files